x/ccv/provider/keeper/throttle.go

```diff
@@ -139,6 +142,8 @@ func (k Keeper) ReplenishSlashMeter(ctx sdktypes.Context) {
 	if meter.GT(allowance) {
 		meter = allowance
 	}
+
+	k.Logger(ctx).Debug("replenish slash meter", "meter", meter.Int64())
```

Comment:
Prob also want to move this after the set method
```suggestion
	k.Logger(ctx).Debug("slash meter replenished", "meter", meter.Int64())
```